src/metabase/pulse/render/table.clj

```diff
@@ -13,24 +14,22 @@
    (style/font-style)
    {:font-size :12px
     :font-weight     700
-    :color           style/color-text-medium
-    :border-bottom   (str "1px solid " style/color-header-row-border)
-    :padding-top     :20px
-    :padding-bottom  :5px}))
+    :color           style/color-text-dark
+    :border-bottom   (str "2px solid " style/color-border)
+    :border-right    0}))
 
 (def ^:private max-bar-width 106)
 
 (defn- bar-td-style []
   (merge
    (style/font-style)
    {:font-size      :12px
-    :font-weight    700
+    :font-weight    400
     :text-align     :left
     :color          style/color-text-dark
-    :border-bottom  (str "1px solid " style/color-body-row-border)
-    :height         :28px
-    :padding-right  :0.375em
-    :padding-left   :0.375em}))
+    :border-bottom  (str "1px solid " style/color-border)
+    :border-right   (str "1px solid " style/color-border)
+    :padding        "0.75em 1em"}))
```

Comment:
Use similar ratio to the design
#### Design
base font size: 22px
horizontal padding: 24px (1.01x the font size) 
vertical padding: 18px (0.82x the font size)

#### Implementation
base font size: 12px
horizontal padding: 1.01x12 = 12.12 (approx. 12px)
vertical padding: 0.82*12 = 9.84 (approx. 10px)

Since the current implementation use relative unit em
- 12/12 = 1em
- But 10/12 = 0.83em which doesn't look good so I reduce the size to 9/12 which result in 0.75em
And that's why I came up with "0.75em 1em" [vertical-padding horrizontal-padding]